src/lib/login/userExists.js

```diff
@@ -0,0 +1,19 @@
+//@flow
+import API from '../../lib/API/api'
+import retryImport from '../../lib/utils/retryImport'
+
+export const userExists = async (mnemonics): Promise<any> => {
```

Comment:
why we need a separate utility for this ? could it be just a method of userStorage which looks (from my side of course, probably you have some reasons for that) more logical ?

Reply:
it uses different classes, so it doesnt really belong in any specific one